### PR TITLE
Feat: add i18n

### PR DESCRIPTION
**Pull request by Thiago-Mota-Santos**
- [x] - Added i18n using Next-i18next

### CODE REVIEW

**Review by daniloab — APPROVED** [review]
why common?

**Comment by Thiago-Mota-Santos** [comment]
> why common?

I followed the docs examples. I can change the name to something more semantic.

File: src/components/integration/add/IndicationAdd.tsx [hunk 1]
```diff
@@ -139,33 +142,31 @@ const IndicationAdd = ({ isEdit, indicationId }: Indication) => {
               clipRule="evenodd"
             />
           </svg>
-          Update
+          {t('update')}
```
Comment (daniloab) [inline src/components/integration/add/IndicationAdd.tsx]: Prefer capitalize words `Update`

File: src/components/integration/add/IndicationAdd.tsx [hunk 2]
```diff
@@ -205,7 +206,7 @@ const IndicationAdd = ({ isEdit, indicationId }: Indication) => {
                 className="w-20 text-right text-base text-violet-400 dark:text-violet-500"
                 htmlFor="apartmentBlock"
               >
-              Bloco AP
+              {t('form-apblock')}
```
Comment (daniloab) [inline src/components/integration/add/IndicationAdd.tsx]: prefer clear keys, avoid names like this one, set all keys in english `Apartment Block`

**Review by daniloab — COMMENTED** [review]
capitalize words and avoid keys with `-`, prefer clear names

**Comment by Thiago-Mota-Santos** [comment]
I improved keys in the json. 

About name json:
```
localeStructure: '{{ns}}'
```

this configuration allows you to change the name, but it is very limited, the names would have to be the same.

image

**Review by daniloab — APPROVED** [review]

**Pull request merged**